Dispatcher/Views/ArticleCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ArticleCell.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 19/04/2023.
+//
+
+import UIKit
+
+class ArticleCell: UITableViewCell {
+    
+    static let articleCellIdentifier = "articleCell"
+    
+
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var tagLabel: UILabel!
+    @IBOutlet weak var autherLabel: UILabel!
+    @IBOutlet weak var subTitleLabel: UILabel!
+    @IBOutlet weak var dispatchButton: UIButton!
+    @IBOutlet weak var moreTagsLabel: UILabel!
+    @IBOutlet weak var addToFavoritesButton: UIButton!
```

Comment:
The button can either add to favorites or remove from favorites. Therefore its name is somewhat misleading and should be changed